server/vcs/markdown/markdown_renderer.go

```diff
@@ -67,10 +67,10 @@ type projectResultTmplData struct {
 	Rendered    string
 }
 
-// Render formats the data into a markdown string.
+// Render formats the data into a markdown string for a command.
 // nolint: interfacer
 func (m *Renderer) Render(res command.Result, cmdName command.Name, baseRepo models.Repo) string {
-	commandStr := strings.Title(strings.Replace(cmdName.String(), "_", " ", -1))
+	commandStr := strings.Title(strings.ReplaceAll(cmdName.String(), "_", " "))
```

Comment:
Is there a case where there is more than 1 `_` in a command?

Reply:
Nope, the only command names with `_` is `policy_check` and `approve_policies` with one `_` only. 
https://github.com/lyft/atlantis/blob/725034498271b90ff59e293eb875060002dcd107/server/events/command/name.go#L41-L44